src/main/java/org/togetherjava/discordbot/commands/commands/ExampleCommandDB.java

```diff
@@ -0,0 +1,36 @@
+package org.togetherjava.discordbot.commands.commands;
+
+import de.ialistannen.commandprocrastination.autodiscovery.ActiveCommand;
+import de.ialistannen.commandprocrastination.command.tree.CommandNode;
+import de.ialistannen.commandprocrastination.parsing.ParseException;
+import org.togetherjava.discordbot.commands.CommandContext;
+import org.togetherjava.discordbot.db.autogen.tables.pojos.Test;
+import org.togetherjava.discordbot.db.repositories.ExampleRepository;
+
+import static de.ialistannen.commandprocrastination.parsing.defaults.StringParsers.greedyPhrase;
+
+@ActiveCommand(name = "example", parentClass = BasePrefixCommand.class)
+@SuppressWarnings("unused")
+public class ExampleCommandDB extends CommandNode<CommandContext> {
```

Comment:
I understand that you want to show how it works but this command should be disabled in the actual bot. It could lead to people abusing it and trying to fill the server harddrive unecessary.